client/app/components/project-geography.js

```diff
@@ -0,0 +1,41 @@
+import Component from '@glimmer/component';
+import EmberObject, { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+
+const bblObject = EmberObject.extend({
+  dcp_validatedlot: '',
+  dcp_bblnumber: '',
+  versionnumber: '',
+  dcp_partiallot: null,
+  dcp_developmentsite: null,
+});
+
+export default class ProjectGeographyComponent extends Component {
+  // object that mimicks the objects that exist in the bbls array
+  // of the pasForm (expand entity: dcp_dcp_projectbbl_dcp_pasform)
+  bblObject = bblObject.create();
+
+  // user-entered bbl number
+  @tracked bblNumberToAdd = '';
+
+  // represents the pasForm model
+  @tracked pasForm;
+
+  @action
+  async addBbl(bblNumber) {
+    // bblObjectToPush will be pushed into bbls array of pasForm
+    const bblObjectToPush = this.bblObject;
+
+    // update the dcp_bblnumber attribute to the user-entered value
+    bblObjectToPush.dcp_bblnumber = bblNumber.toString();
```

Comment:
Question about types, why is this a number in the first place? Is it a number in CRM? Is there a way we can keep it as one thing instead of having to convert?

Reply:
This comes back as a type `number` from labs-search, but dcp_bblnumber in CRM comes back as a string. So I'm setting the bbl that we get from labs-search to string in order to push to the object being posted to CRM

Reply:
cool! makes sense